build/gulpfile.mixin.js

```diff
@@ -6,64 +6,27 @@
 'use strict';
 
 const gulp = require('gulp');
-const json = require('gulp-json-editor');
-const buffer = require('gulp-buffer');
-const filter = require('gulp-filter');
-const es = require('event-stream');
-const util = require('./lib/util');
-const remote = require('gulp-remote-src');
-const zip = require('gulp-vinyl-zip');
-const assign = require('object-assign');
-const pkg = require('../package.json');
+const jeditor = require('gulp-json-editor');
 
 gulp.task('mixin', function () {
```

Comment:
Looking at the old code here, this update code probably belongs in a different task, or at least I'd suggest renaming this task.

Reply:
Why a different task? It still does json mixing, just with a locally defined json for the additions, rather than a remote one.

Reply:
Oh, that makes sense. I guess I was expecting a name specifically related to updates, but that works